share/revng/test/tests/analysis/ImportFromCAnalysis/struct-with-conflicting-annotations/expected-error.txt

```diff
@@ -0,0 +1,7 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import-from-c: Multiple conflicting annotation values found: '8' and '16'.
```

Comment:
`Multiple conflicting _START_AT annotations`

Reply:
This thread is actually the same as the [above one](https://github.com/revng/revng-c/pull/29#discussion_r1734868434), so I'll close it so that the discussion doesn't get split.

Reply:
I'm leaving this open, but I'll continue answering only the other one.

Reply:
Sure, but consider that however we decide to proceed here was always going to affect every single annotation (with a value) anyway.

Reply:
yeah I'm aware